src/libraries/icubmod/usbCamera/linux/V4L_camera.cpp

```diff
@@ -215,14 +215,14 @@ bool V4L_camera::open(yarp::os::Searchable& config)
     // stat file
     if (-1 == stat(param.deviceId.c_str(), &st))
     {
-        fprintf(stderr, "Cannot identify '%s': %d, %s\n", param.deviceId.c_str(), errno, strerror(errno));
+        yError("usbCamera: Cannot identify '%s': %d, %s\n", param.deviceId.c_str(), errno, strerror(errno));
```

Comment:
Suggestion: remove the '\n' at the end of these messages (yError yInfo yDebug already insert a linebreak).